spec/system/support/prior_authority/step_helpers.rb

```diff
@@ -64,6 +64,7 @@ def fill_in_case_contact
     end
 
     def fill_in_client_detail
+      click_on 'Client details'
```

Comment:
i did not realise you were working on this and started a [branch this am to handle the flow generally](https://github.com/ministryofjustice/laa-claim-non-standard-magistrate-fee-backend/pull/507).

Having looked into this a bit I found that having the click_on in here made the individuals spec less flexible in testing the checks for tasklist status e.g. "Not started", plus just from a convention point of view I think the fill_in... methods should only fill_in the fields (not perform navigation (to avoid confusing our future selves. I therefore added the click_on  in the fill_in_until_step method.

Reply:
Good point about `click_on`. Ok, how shall we divvy this up? I'm working on this to get CRM457-975 to pass QA. Do we want a single branch that handles both:
- The new behaviour for 'Save and come back later'
- Standardising returning to the task list after completing a task?

Reply:
Ok @jsugarman I've updated to move the 'click_on's out of the 'fill_ins'